app/src/controllers/modal/actionCreators.js

```diff
@@ -1,10 +1,10 @@
 import { SHOW_MODAL, HIDE_MODAL } from './constants';
 
 
-export const showModalAction = ({ modalType }) => ({
+export const showModalAction = ({ modalId, modalData }) => ({
```

Comment:
`{ id, data }` - there's no need to have different names

Reply:
fixed